test/unit/test_converters.py

```diff
@@ -0,0 +1,70 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test Util Converters."""
+
+from datetime import datetime
+import pytz
+from qiskit_ibm_provider.utils import converters
+from ..ibm_test_case import IBMTestCase
+
+
+class TestConverters(IBMTestCase):
```

Comment:
local_to_utz_str, convert_tz and str_to_utz methods are not used anywhere and hence those methods can be removed.